### ISSUE TITLE
[NPC] Deathguard Terrence, emote issue

### ISSUE THREAD

**Issue by Richteras** [dalaranwow/dalaran-wow#5090]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The NPC Deathguard Terrace that patrols outside of Brill is constantly laughing. 
**Expected behaviour**: Tell us what should happen instead
If memory serves, he should say something, laugh once, and continue with the patrol. This isn't a major bug just a very slightly annoying one. 
**Steps to reproduce the problem**:

1. Stand outside the Brill Inn for a couple on minutes
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Richteras.
Issue confirmed on PTR, he is using emote state laught instead of oneshot laught.

![wowscrnshot_112217_131318](https://user-images.githubusercontent.com/6797611/33125976-5e37495a-cf8b-11e7-988d-26dbaef058e6.jpg)

Note: Creature uses waypoint script.

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3828
